packages/expo-processing/index.js

```diff
@@ -57,28 +54,23 @@ export class ProcessingView extends React.Component {
       loc = origGetAttribLocation.call(gl, program, name);
       program.attribLocationCache[name] = loc;
       return loc;
-    }
-
+    };
 
     // Call `gl.endFrameEXP()` every frame
 
     const keepFlushing = () => {
       gl.endFrameEXP();
-      requestAnimationFrame(keepFlushing);
-    }
+      this.rafID = requestAnimationFrame(keepFlushing);
```

Comment:
If it's not intended to be public:

```suggestion
      this._rafID = requestAnimationFrame(keepFlushing);
```